front/package.json

```diff
@@ -26,16 +29,33 @@
     "wagmi": "^2.12.17"
   },
   "devDependencies": {
+    "@chromatic-com/storybook": "^1.9.0",
+    "@storybook/addon-essentials": "^8.3.5",
+    "@storybook/addon-interactions": "^8.3.5",
+    "@storybook/addon-links": "^8.3.5",
+    "@storybook/addon-onboarding": "^8.3.5",
+    "@storybook/blocks": "^8.3.5",
+    "@storybook/nextjs": "^8.3.5",
+    "@storybook/react": "^8.3.5",
+    "@storybook/test": "^8.3.5",
     "@types/node": "^20.16.11",
     "@types/react": "^18.3.11",
     "@types/react-dom": "^18.3.0",
     "eslint": "^8.57.1",
     "eslint-config-next": "14.2.15",
     "eslint-plugin-perfectionist": "^3.8.0",
+    "eslint-plugin-storybook": "^0.9.0",
     "postcss": "^8.4.47",
     "prettier": "^3.3.3",
+    "storybook": "^8.3.5",
     "tailwindcss": "^3.4.13",
-    "typescript": "^5.6.3"
+    "typescript": "^5.6.3",
+    "webpack": "^5.95.0"
```

Comment:
Webpack 🤮, why?

Reply:
Hard dependency of storybook, will not be used/included in the front bundle though